discovery/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang
```

Comment:
This shouldn't be in the `discovery/` dir, but `cluster/images/jws-discovery` or something like that.

Also, it should be 
```
FROM busybox
COPY kube-discovery /usr/local/bin
ENTRYPOINT ["/usr/local/bin/kube-discovery"]
````

in order to be consistent with the other server images